src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let callCount = 0;
+
+  return (a) => {
```

Comment:
It is not required for this task but you can give default values for a and b so if no arguments are passed. your code still works correctly.

Reply:
yep @lefteris-L 
@ysrckr is right, i'd provide the default values, because what would then be a result of 
`undefinend + undefined`?